e2e/various-suite/explore.spec.ts

```diff
@@ -11,41 +11,51 @@ e2e.scenario({
     e2e.pages.Explore.General.container().should('have.length', 1);
     e2e.components.RefreshPicker.runButtonV2().should('have.length', 1);
 
-    e2e.components.DataSource.TestData.QueryTab.scenarioSelectContainer()
-      .should('be.visible')
-      .within(() => {
-        e2e().get('input[id*="test-data-scenario-select-"]').should('be.visible').click();
-      });
+    e2e.components.QueryTab.queryHistoryButton().should('be.visible').click();
 
-    cy.contains('CSV Metric Values').scrollIntoView().should('be.visible').click();
-
-    cy.location().then((loc) => {
-      const params = new URLSearchParams(loc.search);
-      const leftJSON = JSON.parse(params.get('left'));
-      expect(leftJSON.range.to).to.equal('now');
-      expect(leftJSON.range.from).to.equal('now-1h');
-
-      cy.get('body').click();
-      cy.get('body').type('t{leftarrow}');
-
-      cy.location().then((locPostKeypress) => {
-        const params = new URLSearchParams(locPostKeypress.search);
-        const leftJSON = JSON.parse(params.get('left'));
-        // be sure the keypress affected the time window
-        expect(leftJSON.range.to).to.not.equal('now');
-        expect(leftJSON.range.from).to.not.equal('now-1h');
-        // be sure the url does not contain dashboard range values
-        // eslint wants this to be a function, so we use this instead of to.be.false
-        expect(params.has('to')).to.equal(false);
-        expect(params.has('from')).to.equal(false);
+    e2e.components.QueryHistory.queryText()
+      .should('be.visible')
+      .then((initialQueryHistoryEntries) => {
+        e2e.components.QueryTab.queryHistoryButton().should('be.visible').click();
```

Comment:
the preview is kinda misleading, these are the only changed line that are needed open the query history tab, get the initial number of queries (used then in the assertions on line 57) and close the the drawer